sympy/geometry/polygon.py

```diff
@@ -2217,6 +2220,53 @@ def incircle(self):
         return Circle(self.incenter, self.inradius)
 
     @property
+    def exradii(self):
+        """The radius of excircles of a triangle.
+
+        An excircle of the triangle is a circle lying outside the triangle,
+        tangent to one of its sides and tangent to the extensions of the
+        other two.
+
+        Returns
+        =======
+
+        exradii : dict
+
+        See Also
+        ========
+
+        sympy.geometry.polygon.Triangle.inradius
+
+        Examples
+        ========
+
+        >>> from sympy.geometry import Point, Triangle, Segment2D, Point2D
+        >>> p1, p2, p3 = Point(0, 0), Point(6, 0), Point(0, 2)
+        >>> t = Triangle(p1, p2, p3)
+        >>> t.exradii[Segment2D(Point2D(0, 0), Point2D(0, 2))]
```

Comment:
My intention was to let the key be the vertex opposite of the segment that is tangent to the excircle, so the excircle on the segment between vertex 0 and 1 would be keyed to vertex 2. But I guess it's not too different to write `t.exradii[t.sides[0]]` instead of `t.exradii[t.vertex[2]]`. Do you mind writing the doctest as `>>> t.exradii[t.sides[2]]`?

Reply:
Perhaps could add some text so it reads:

```
The exradius touches the side of the triangle to which it is keyed, e.g. the exradius
touching side 2 is:

>>> t.exradii[t.sides[2]]
-2 + sqrt(10)
```

>>>

Reply:
Sure thing - I will rewrite the doctest and add some text.